app/controllers/logged_area_controller.rb

```diff
@@ -0,0 +1,31 @@
+require 'private_beta_user'
+
+class LoggedAreaController < ApplicationController
+  before_action :authenticate
+end
+
+def authenticate
+  return if Rails.env.development? || Rails.env.test?
```

Comment:
If it is part of the specification of the app that the user must be logged in, then I'd suggest writing a test helper that signs a user in, and then use that helper in all test that require it.